github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/context/GitHubPRTriggerDslContext.java

```diff
@@ -33,6 +35,14 @@ public void setPreStatus() {
         setPreStatus = true;
     }
 
+    public void setCancelQueued() {
```

Comment:
The idea of DSL to have human readable single commands i.e. "stage" and not "runStage()", `scm` and not `checkoutScm`. But yeah, it can be interpreted in both ways. Having all DSL starting with set seems not good also as we want to have DSL and not POJO. I think you can compare with already existed dsls.